stdlib/in_channel.mli

```diff
@@ -165,3 +180,19 @@ val isatty : t -> bool
     [false] otherwise.
 
     @since 5.1 *)
+
+(** {1:examples Examples}
```

Comment:
Maybe some people think otherwise but as far as I'm concerned I'm not very fond of having toplevel interactions as examples. I would rather have functions you can cut and paste in your toplevel to play with.

Reply:
I was basing the examples off the [queue.mli](https://github.com/ocaml/ocaml/blob/34cf5aafcedc2f7895c7f5f0ac27c7e58e4f4adf/stdlib/queue.mli#L124), but yeah, I agree that they don't really add much in this context, so removing the toplevel comments.